### DISCUSSION TITLE
Is there a way to customise the modal form layout used by the exporter action?

### DISCUSSION THREAD

**Discussion by markdynamo6** [filamentphp/filament#14413] [Help]
### Package

Panel builder

### Package Version

^v3.2

### How can we help you?

The ExportColumn array is quite a long for one of our tables and it would look better with a 2 column layout. But it's not obvious how to customise the 'Columns' fieldset that these ExportColumns display in.

**Comment by MedtheVorg** [comment]
Hi 
I was able to use css to override the modal style and this will only work fine as long as you do not have any other custom modal with a fieldset inside,  because all the modals use the same classes and I could  not find any way to add a css class to the exporter modal only.

css : 

```
.fi-modal-window:has(.fi-modal-content fieldset div) {
  max-width: 64rem;
}

.fi-modal-window .fi-modal-content fieldset div {
  --cols-lg: repeat(4, minmax(0, 1fr)) !important;
  max-height: 600px;
  overflow: auto;
  gap: 1rem !important;
}

@media (min-width: 640px) {
  .fi-modal-window .fi-modal-content fieldset div div div div:nth-child(2) div div {
  
    grid-template-columns: repeat(1, minmax(0, 1fr));

    }
  
}
```

by the way this css was added to a custom  filament panel theme.css file
